res/css/photon/button.css

```diff
@@ -4,9 +4,12 @@
 
 /* See https://design.firefox.com/photon/components/buttons.html for the spec */
 .photon-button {
-  /* These two flex options aren't necessary when a real <button> is used, but they are when a <a> element is used as abutton */
-  display: flex;
+  /* These flex and sizing properties aren't necessary when a real <button> is used,
+   * but they are when a <a> element is used as a button. */
+  display: inline-flex;
+  box-sizing: border-box;
```

Comment:
buttons are `border-box` by default, while links are `content-box`. So this makes them even.